src/apps/common/ns_turn_utils.c

```diff
@@ -543,30 +545,42 @@ void turn_log_func_default(char *file, int line, TURN_LOG_LEVEL level, const cha
     so_far += snprintf(s, sizeof(s), "%lu: ", (unsigned long)log_time());
   }
 
+#if defined(WINDOWS) || defined(__CYGWIN__) || defined(__CYGWIN32__) || defined(__CYGWIN64__)
+  so_far += snprintf(s + so_far, MAX_RTPPRINTF_BUFFER_SIZE - (so_far + 1), "[%lu:%lu] ", GetCurrentProcessId(),
+                     GetCurrentThreadId());
+#else
+
 #ifdef SYS_gettid
-  so_far += snprintf(s + so_far, MAX_RTPPRINTF_BUFFER_SIZE - (so_far + 1), "(%lu): ", (unsigned long)gettid());
+  so_far += snprintf(s + so_far, MAX_RTPPRINTF_BUFFER_SIZE - (so_far + 1), "[%lu] ", (unsigned long)gettid());
```

Comment:
This will change existing log format. People who rely on it - it will break for them (do not know if there are any).

Reply:
I see there are many (good) format changes. I suppose we will need to announce logging format breaking changes in the next version @ggarber

Reply:
Agree! We can use the existing open source logging system. eg: log4cplus